drivers/media/xen/xen_camera_front_cfg.c

```diff
@@ -37,9 +37,14 @@ static int cfg_read_framerates(struct xenbus_device *xb_dev,
 		return PTR_ERR(list);
 	}
 
-	/* Make sure the list is not empty. */
+	/*
+	 * Empty list means the "streaming parameters" such as frame rate
+	 * are not supported for underlying device at the backend side, therefore
```

Comment:
This could easily be that we just do not configure frame rates for the given guest, but backend still may support this

Reply:
ok, will update